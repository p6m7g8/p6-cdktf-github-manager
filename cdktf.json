{
  "language": "typescript",
  "app": "npx ts-node src/main.ts",
  "projectId": "b5ff9e8e-b76c-4b95-a6bb-427ea87e59f4",
  "sendCrashReports": "false",
  "terraformProviders": [],
  "terraformModules": [],
  "context": {}
}