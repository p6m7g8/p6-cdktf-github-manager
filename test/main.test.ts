import { TerraformStack } from 'cdktf'
import { Testing } from 'cdktf/lib/testing'
import 'cdktf/lib/testing/adapters/jest'

describe('unit testing using snapshots', () => {
  it('tests the snapshot', () => {
    const app = Testing.app()
    const stack = new TerraformStack(app, 'test')

    expect(Testing.synth(stack)).toMatchSnapshot()
  })
})
