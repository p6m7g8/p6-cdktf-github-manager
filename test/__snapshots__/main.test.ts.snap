// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unit testing using snapshots tests the snapshot 1`] = `
"{
}"
`;
